text/src/autogluon/text/automm/data/process_image.py

```diff
@@ -269,17 +273,23 @@ def construct_processor(
             elif trans_mode == "vertical_flip":
                 processor.append(transforms.RandomVerticalFlip())
             elif trans_mode == "color_jitter":
-                if args is not None:
+                if kargs is not None:
+                    processor.append(transforms.ColorJitter(**kargs))
+                elif args is not None:
                     processor.append(transforms.ColorJitter(*args))
                 else:
                     processor.append(transforms.ColorJitter(brightness=0.1, contrast=0.1, saturation=0.1))
             elif trans_mode == "affine":
-                if args is not None:
+                if kargs is not None:
+                    processor.append(transforms.RandomAffine(**kargs))
+                elif args is not None:
                     processor.append(transforms.RandomAffine(*args))
                 else:
-                    processor.append(transforms.RandomAffine(15, translate=(0.1, 0.1), scale=(0.9, 1.1)))
+                    processor.append(transforms.RandomAffine(degrees=15, translate=(0.1, 0.1), scale=(0.9, 1.1)))
             elif trans_mode == "randaug":
-                if args is not None:
+                if kargs is not None:
+                    processor.append(RandAugment(**kargs))
+                elif args is not None:
                     processor.append(RandAugment(*args))
                 else:
                     processor.append(RandAugment(2, 9))
```

Comment:
The if-else patterns of `args` and `kwargs` repeat for each augmentation type. Consider make this pattern a separate util function. The default values can also be handled by the `kwargs` choice. For example, if we detect augmentation type `affine`, we can create a default `kwargs` {"degree"=15, "translate"=(0.1, 0.1), "scale"=(0.9, 1.1)} for it.

Reply:
May be we can do it in a separate PR.